### ISSUE TITLE
 Generated keys not requested

### ISSUE THREAD

**Issue by LGMAM** [jcabi/jcabi-jdbc#49]
Just starting to use this library, so maybe some silly error I am not able to detect

```
sending_id = session
                .autocommit(false)
                .sql("START TRANSACTION")
                .execute()
                .sql("INSERT INTO table (name,stamp,username,user_id) VALUES (?,?,?,?)")
                .set("ws").set(0).set(0).set(0)
                .insert(Outcome.LAST_INSERT_ID);
            session.commit();
```

Throws this error

```
Caused by: java.sql.SQLException: Generated keys not requested. You need to specify Statement.RETURN_GENERATED_KEYS to Statement.executeUpdate(), Statement.executeLargeUpdate() or Connection.prepareStatement().
    at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957)
    at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896)
    at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885)
    at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860)
    at com.mysql.jdbc.StatementImpl.getGeneratedKeys(StatementImpl.java:1682)
    at com.jolbox.bonecp.StatementHandle.getGeneratedKeys(StatementHandle.java:672)
    at com.jcabi.jdbc.Request$1.fetch(Request.java:55)
    at com.jcabi.jdbc.JdbcSession.run(JdbcSession.java:378)
    ... 12 more
```

Any hint ?

Thanks.

**Comment by dmarkov** [comment]
@yegor256 please pay attention to this issue ([par.21](http://at.teamed.io/policy.html#21))

**Comment by yegor256** [comment]
@LGMAM first of all, your code should look like this:

```java
insert = session
  .autocommit(false)
  .sql("START TRANSACTION")
  .execute()
  .sql("INSERT INTO table (name,stamp,username,user_id) VALUES (?,?,?,?)")
  .set("ws").set(0).set(0).set(0);
id = insert.insert(Outcome.LAST_INSERT_ID);
insert.commit();
```

Try it and let me know if works...

**Comment by nshusa** [comment]
@yegor256 I get this issue as well only on mysql. Postgres works perfectly.

**Comment by yegor256** [comment]
@nshusa can you show your stacktrace please

**Comment by andreoss** [comment]
Fixed in #95. Added more tests in #97.
